worker/index.js

```diff
@@ -17,7 +17,7 @@
  */
 'use strict'; // eslint-disable-line strict
 const featureToggles = require('feature-toggles');
-const logger = require('winston');
+const logger = require('../logger');
```

Comment:
Need to init the producer here as well, since this is an entry point for a separate process.